src/core/core.helpers.js

```diff
@@ -430,11 +430,13 @@ module.exports = function() {
 		return value !== undefined && value !== null && value !== 'none';
 	}
 
-	// Private helper to get a constraint dimension
-	// @param domNode : the node to check the constraint on
-	// @param maxStyle : the style that defines the maximum for the direction we are using (maxWidth / maxHeight)
-	// @param percentageProperty : property of parent to use when calculating width as a percentage
-	// @see https://www.nathanaeljones.com/blog/2013/reading-max-width-cross-browser
+	/**
+	 * Private helper to get a constraint dimension
+	 * @param domNode - the node to check the constraint on
+	 * @param maxStyle - the style that defines the maximum for the direction we are using (maxWidth / maxHeight)
+	 * @param percentageProperty - property of parent to use when calculating width as a percentage
+	 * @see https://www.nathanaeljones.com/blog/2013/reading-max-width-cross-browser
+	 */
```

Comment:
If we're going to start a new rule that we're banning comments on private methods (I'm not sure I agree) then I'd prefer to discuss amongst all the reviewers and then send one PR that implements across the whole code base and adds to the contributor's guide . That's really outside the scope of this PR though

Reply:
Just saying that instead of reformatting this useless comment, we could just keep the only useful information: the URL. I'm not asking to remove all private jsdoc comments *in this PR*, and agree that this point should be discussed with other collaborators before going this direction. Though, that's not a priority.

> ... a new rule that we're banning comments on private methods ...

To be clear: I'm not suggesting to ban comments on private code (we sometime need the "why"), I'm suggesting to stop using complex / detailed jsdoc for private code in favor of self documented code. Still, that's outside the scope of this PR.

Reply:
I found what `maxStyle` was expecting as a parameter to be pretty helpful. Though it wasn't a great comment overall. I updated it a bit to make it more useful

Reply:
Fine